pkg/api/types.go

```diff
@@ -1305,8 +1305,9 @@ type Endpoints struct {
 //     a: [ 10.10.1.1:8675, 10.10.2.2:8675 ],
 //     b: [ 10.10.1.1:309, 10.10.2.2:309 ]
 type EndpointSubset struct {
-	Addresses []EndpointAddress
-	Ports     []EndpointPort
+	Addresses         []EndpointAddress
+	NotReadyAddresses []EndpointAddress
```

Comment:
Not in the same subset.  We split the subset

On Sep 15, 2015, at 8:36 PM, Tim Hockin <notifications@github.com> wrote:

In pkg/api/types.go
<https://github.com/kubernetes/kubernetes/pull/13778#discussion_r39583170>:

> @@ -1305,8 +1305,9 @@ type Endpoints struct {
>  //     a: [ 10.10.1.1:8675, 10.10.2.2:8675 ],
>  //     b: [ 10.10.1.1:309, 10.10.2.2:309 ]
>  type EndpointSubset struct {
> -	Addresses []EndpointAddress
> -	Ports     []EndpointPort
> +	Addresses         []EndpointAddress
> +	NotReadyAddresses []EndpointAddress

Is this a one-of situation? Is it valid to have Addresses and
NotReadyAddresses at the same time in a single subset?

Comment pls.

—
Reply to this email directly or view it on GitHub
<https://github.com/kubernetes/kubernetes/pull/13778/files#r39583170>.